borderland-tool/borderland_tool/pretix/__init__.py

```diff
@@ -4,27 +4,30 @@
 import math
 
 class PretixAPI:
-    def __init__(self, host, org, event, token):
+    def __init__(self, host, org, event, token, no_ssl=False):
         self.host = host
         self.org = org
         self.event = event
         self.token = token
+        self.url_scheme = "https"
+        if no_ssl:
+            self.url_scheme = "http"
 
 
     def get_orders(self, status="p"):
-        url = "https://{}/api/v1/organizers/{}/events/{}/orders/?page=1".format(self.host, self.org, self.event)
+        url = f"{self.url_scheme}://{self.host}/api/v1/organizers/{self.org}/events/{self.event}/orders/?page=1"
```

Comment:
looks good

Reply:
perhaps factor out the part before until {self.event}, it repeats in many parts of the code. Would avoid one-off errors

Reply:
yep, that's something that we could do later